bode/bode/models/task.py

```diff
@@ -35,6 +36,12 @@ class Task(db.Model):
 
     tags = db.relationship("Tag", secondary=task_tag, back_populates="task")
 
+    @hybrid_property
+    def relation_types(self):
+        from bode.models.task_relations_actions import get_relation_types
```

Comment:
> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.

See: https://peps.python.org/pep-0008/#imports

Reply:
And if it is a loop import, then it means that our file dependency structure is not a tree but has a cycle. It would be a pain to maintain this code in the future if it is so. In this case, we need to refactor the file structure so that there's no deps cycle.

Reply:
Yeah, we need a refactor of our file structure. We should take out almost all db querying and updating code from models and put it elsewhere and that should help with circular imports. I wouldn't do that in this PR though so ITG-104 is blocked for too long. So I'd leave these imports for now but go for the refactor ASAP in another PR.